openshift-examples/keycloak-https-mutual-tls.json

```diff
@@ -212,6 +218,14 @@
                                     {
                                         "name": "X509_CA_BUNDLE",
                                         "value": "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+                                    },
+                                    {
+                                        "name": "JGROUPS_DISCOVERY_PROTOCOL",
+                                        "value": "dns.DNS_PING"
+                                    },
+                                    {
+                                        "name": "JGROUPS_DISCOVERY_PROPERTIES",
+                                        "value": "${APPLICATION_NAME}.${NAMESPACE}.svc.cluster.local"
```

Comment:
Shouldn't the `value` be `dns_query=${APPLICATION_NAME}.${NAMESPACE}.svc.cluster.local`?

Reply:
At the moment, no. The `svc.cluster.local` is appended automatically. However, we consider this as a mistake and there's already a [JIRA](https://issues.jboss.org/browse/JGRP-2295) to remove it.

Reply:
This seems wrong, it should be `"value": "dns_query=${APPLICATION_NAME}.${NAMESPACE}.svc.cluster.local"`